src/main/java/anichan/parser/WorkspaceParser.java

```diff
@@ -12,13 +12,18 @@
  */
 public class WorkspaceParser extends CommandParser {
     private static final Logger LOGGER = AniLogger.getAniLogger(BookmarkParser.class.getName());
+    public static final String ASSERTION_INVALID_MESSAGE = "Input should not be null.";
     public static final String EXCEPTION_INVALID_PARAMETERS = "Invalid parameters detected!";
+    public static final String EXCEPTION_ILLEGAL_WORKSPACE_NAME = "Workspace name must be shorter than "
+            + "31 characters and consist only alphanumeric characters and/or spaces.";
+
     public static final String COMMAND_NEW = "n";
     public static final String COMMAND_SWITCH = "s";
     public static final String COMMAND_list = "l";
     public static final String COMMAND_LIST = COMMAND_list;
     public static final String COMMAND_DELETE = "d";
     public static final String REGEX_SPACE_CHARACTER = " ";
+    public static final int MAXIMUM_WORKSPACE_NAME_LENGTH = 30;
```

Comment:
Similarly here, is there a reason for these to be public?

Reply:
Thanks for the alert, shall fix these!